src/Angor/Client/Pages/Signatures.razor

```diff
@@ -32,137 +32,190 @@
     NavigationManager.NavigateTo($"/wallet");
     return;
 }
-<div class="row">
+
+<div class="row slide-in">
```

Comment:
there will be some changes to this page in the PR that will go in before this, you may need to fix conflicts